### PR TITLE
Fix CodeQL default setup docs

### PR DESCRIPTION
**Pull request by FidelusAleksander**
### Why:

If there are no supported languages found in the repository then the CodeQL setup is not available. The documentation seems  wrong.

Closes: 

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline (this link will be available after opening the PR).

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [x] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Comment by nguyenalex836** [comment]
@FidelusAleksander Thanks so much for opening a PR! I'll get this triaged for review ✨

**Review by sabrowning1 — CHANGES_REQUESTED** [review]
👋🏼 @FidelusAleksander, thanks for opening this PR! You're right, this content could use some clarification. I've requested some changes below; let me know when you've taken a look and I'll give this another review 🙂

File: content/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ For {% data variables.product.product_name %}, before configuring {% data variab
 Enterprise owners, organization and repository administrators can add self-hosted runners. For more information, see "[AUTOTITLE](/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners)" and "[AUTOTITLE](/actions/hosting-your-own-runners/managing-self-hosted-runners/adding-self-hosted-runners)."
 {% endif %}
 
-You can use default setup if your repository includes languages that aren't supported by {% data variables.product.prodname_codeql %}, such as R. For more information on {% data variables.product.prodname_codeql %}-supported languages, see "[AUTOTITLE](/code-security/code-scanning/introduction-to-code-scanning/about-code-scanning-with-codeql#about-codeql)."
+You can use default setup if your repository includes languages that are supported by {% data variables.product.prodname_codeql %}, such as Python or Java. For more information on {% data variables.product.prodname_codeql %}-supported languages, see "[AUTOTITLE](/code-security/code-scanning/introduction-to-code-scanning/about-code-scanning-with-codeql#about-codeql)."
```
[the remark this answers is not in the document]
Comment (FidelusAleksander) [reply content/code-security/code-scanning/enabling-code-scanning/configuring-default-setup-for-code-scanning.md]: LGTM @sabrowning1 !

**Review by sabrowning1 — APPROVED** [review]
This looks great @FidelusAleksander! I'll merge this shortly. Thanks again for your help maintaining our docs! 🚀

**Pull request merged**